src/univers/version_constraint.py

```diff
@@ -48,6 +56,7 @@ def operator_star(a, b):
     ">": operator.gt,
     "=": operator.eq,
     "*": operator_star,
+    "^": operator_null,
```

Comment:
Do we really want to introduce this new syntax? Please explain the rationale and need in detail in a comment or in PR comment.